talent/application-process.md

```diff
@@ -0,0 +1,68 @@
+
+### Screening
+
+- In less than a week after the submission of the application the developer receives an email with feedback, however there can be delays. In any case, the developer is free to reach out to us at any moment at talent@sourced.tech.
+
+- In case of positive feedback in the application, the developer is asked to move to a quick Skype/Zoom/Phone call with the lead of the potential hiring team.
+
+### Skype/Zoom/Phone call
+
+- The developer has a quick call with the lead engineer of the team he applied to. This should be a general discussion that can last between 15-30 minutes. Its purpose is to know the background and experience of the developer, as well as the developer to know source{d} better. It's expected that the developer has already done a basic research about source{d} and has specific questions about the role and/or the company to be expressed during the call. 
+
+- There are notable exceptions to this, such as higher volume junior roles where we directly ask the developer to do the coding challenge before a call.
+
+### Coding Challenge
+
+- Every developer who passes the first interview should complete a coding challenge remotely.
+
+- The coding challenge can be solved in Go/Python/JavaScript/Scala/Ruby.
+
+- Unit tests are provided to validate the correctness of the solution.
+
+- The time limit for solving the coding challenge is 5 hours. 
+
+- We are expecting the solution either in a private Gist or in an attached file.
+
+- The main point being reviewed is the correctness of the solution, however we do check the complexity and code style. 
+
+- In case of the Data Science team, there is an additional ML task necessary to be solved in the time limit of 5 hours. 
+
+- The results are reviewed by the engineering team and feedback is given to the developer as soon as possible. 
+
+
+### On-site Interview
+
+- Every developer who successfully solves the coding challenge, is invited for the last interview at our offices in Madrid. It might last a full day or be split across two half-days, depending on the availability of the developer.
+
+- This is the last step in which it's decided if the developer will receive an offer or not. 
+
+The interviews are the following:
+
+- Kickoff with CTO, QA: to know the developer better.
+
+- Design interview: to see how the developer would design a solution to a given technical problem working with a several of our engineers.
+
+- Logical thinking: to see how the developer solves an abstract problem in a logical way.
+
+- In case of rejection, the developer should be notified in person on the day by the lead of the corresponding team. This should be a constructive discussion giving the developer as much insight as possible. 
+
+### Offer
+
+- The offer is being sent by CTO on Lever. 
+
+- The developer has no time deadline to accept/reject the offer. Only in cases that we have more developers in the offer state for one opening where decisions need to be made. 
```

Comment:
In cases where we have a limited # of open positions for a role, and a larger number of people we'd like to make an offer too, a time limit is given.